src/sonic-yang-models/yang-models/sonic-telemetry.yang

```diff
@@ -74,5 +74,27 @@ module sonic-telemetry {
             }
 
         }
+
+        container TELEMETRY_CLIENT_CERT {
+            description "TELEMETRY client cert list";
+
+            list TELEMETRY_CLIENT_CERT_LIST {
```

Comment:
>TELEMETRY_CLIENT_CERT_LIST [](http://example.com/codeflow?start=17&length=26)

similar to GNMI_CLIENT_CERT_LIST. Can we reuse?

Reply:
Fixed, reuse GNMI_CLIENT_CERT table for both GNMI and Telemetry service